core/src/style/pointStyle.cpp

```diff
@@ -15,6 +15,8 @@
 #include "data/propertyItem.h" // Include wherever Properties is used!
 #include "scene/stops.h"
 
+constexpr float texture_scale = static_cast<float>(std::numeric_limits<int16_t>::max());
```

Comment:
Feels a little odd to use a literal for this scale in one place and not in the other (even though I know it's the same value)